aws/rust-runtime/aws-config/src/profile/credentials/repr.rs

```diff
@@ -473,7 +473,7 @@ mod tests {
     use crate::sensitive_command::CommandWithSensitiveArgs;
     use serde::Deserialize;
 
-    #[cfg(feature = "test-utils")]
+    #[cfg(feature = "test-util")]
```

Comment:
Is this just a name change without any programmatic effects?

Reply:
Beyond the name change, the `aws-config` crate doesn't seem to have a `test-util(s)` feature listed in `Cargo.toml` at all so I'm not sure how this is working. Does it transitively work with the `test-util` feature present in some of the other crates?

Reply:
We never added these features to the cargo.toml so I'm pretty sure they were never usable. I changed the name to bring it in line with the other crates. I'm also going to add the feature to the `Cargo.toml`.

Reply:
I'm going to transfer this change to a separate PR because it's causing some test failures.